static/_app/tools/dadish-2/gamebridge.js

```
@@ -0,0 +1,116 @@
+function a1_0xc7ff() {
```

Biome (1.9.4)

[error] Do not reassign a function declaration.

Reassigned here.

Use a local variable instead.

(lint/suspicious/noFunctionAssign)